ansible/roles/sysprep/tasks/main.yml

```diff
@@ -224,3 +224,15 @@
   when:
     - disable_firewalld | default(true)
     - firewalld_exists.rc == 0
+
+- name: Ensure containterd is started
+  systemd:
+    name: containerd
+    enabled: true
+    state: started
+
+- name: Ensure kubelet is started
+  systemd:
+    name: containerd
```

Comment:
```suggestion
    name: kubelet
```

Reply:
Whoops, bad copy pasta. Good catch. Fixed and repushed. But since we're talking about it, should we just remove the starting of `kubelet` in KIB. It will fail to start since there is no config, and the `goss` checks just check that it is present and either started or stopped.